src/drivers/heater/heater.h

```diff
@@ -163,9 +164,32 @@ class Heater : public ModuleBase<Heater>, public ModuleParams, public px4::Sched
 	 */
 	void update_params(const bool force = false);
 
+	/**
+	 * @brief Enables / configures the heater (either by GPIO or PX4IO)
+	 */
+	void heater_enable(void);
```

Comment:
```suggestion
	void heater_enable();
```

Nitpicky, but clang-tidy will complain.